### ISSUE TITLE
Environment variables reference list

### ISSUE THREAD

**Issue by larsnystrom** [soketi/docs#5]
Hi,

Thank you for Soketi, it looks great.

I'm struggling to find a list of all available environment variables for soketi. As an example, I'm trying to change the port soketi listens on by default. I think it listens to 127.0.0.1:6001 inside a docker container, which makes it impossible to connect to from outside the container. This is probably pretty easy to solve if I can just find the correct environment var to change. But the general problem here is that I don't know which knobs there are to tweak.

Thank you for your time.

**Comment by rennokki** [comment]
I agree, there should be some reference list for all the configurable variables.

For your use case, it's simply `PORT` and you should change the `-p` Docker argument.

**Comment by ProgJo** [comment]
Hi,
I was looking for the same thing and found them here after some digging and studying : 
[https://github.com/soketi/soketi/blob/master/src/cli/cli.ts](https://github.com/soketi/soketi/blob/master/src/cli/cli.ts)

You will have to prefix them with **SOKETI_** like following in your `.env`  file: 
```
SOKETI_APP_MANAGER_DRIVER=appManagerDriver
SOKETI_CACHE_DRIVER='cacheDriver'
SOKETI_DEFAULT_APP_ID=sdf456ert89654
```

And I believe that the format follows a normal `.env` file.

Hope that I'm not mistaken and that helps. But still, an official list would be better.

**Comment by hoelderlin2368** [comment]
I am struggling to understand in general how to set the environment variables correctly on Ubuntu 22.04 here https://docs.soketi.app/getting-started/environment-variables and here https://docs.soketi.app/app-management/sql-drivers/mysql

I am trying to set the MySQL environment variables on my system but I feel it keeps ignoring the settings. Or at least I don't see anything in the logs. Soketi does start successfully and I can reach the websocket ports but how I do I know it reads the environment variables I set?

I also tried to create an env file but this is also poorly documented so I don't understand what goes in the file and where it is saved. Can somebody please explain step by step how to make the settings work that are described in the documentation, especially when SQL drivers are required.

See my example I tried to set in the system variables:

/$ env

SHELL=/bin/bash
SOKETI_APP_MANAGER_DRIVER=mysql
SOKETI_DB_MYSQL_PASSWORD=mydbpassword
PWD=/
SOKETI_DB_MYSQL_DATABASE=mydb
LOGNAME=localuser
XDG_SESSION_TYPE=tty
DEFAULT_APP_SECRET=12345
SOKETI_DB_MYSQL_PORT=3306
MOTD_SHOWN=pam
SOKETI_DB_MYSQL_HOST=192.168.100.100
HOME=/home/localuser
LANG=C.UTF-8
LS_COLORS=rs=0:di=01;34:ln=01;36:mh=00:pi=40;33:so=01;35:do=01;35:bd=40;33;01:cd=40;33;01:or=40;31;01:mi=00:su=37;41:sg=30;43:ca=30;41:tw=30;42:ow=34;42:st=37;44:ex=01;32:*.tar=01;31:*.tgz=01;31:*.arc=01;31:*.arj=01;31:*.taz=01;31:*.lha=01;31:*.lz4=01;31:*.lzh=01;31:*.lzma=01;31:*.tlz=01;31:*.txz=01;31:*.tzo=01;31:*.t7z=01;31:*.zip=01;31:*.z=01;31:*.dz=01;31:*.gz=01;31:*.lrz=01;31:*.lz=01;31:*.lzo=01;31:*.xz=01;31:*.zst=01;31:*.tzst=01;31:*.bz2=01;31:*.bz=01;31:*.tbz=01;31:*.tbz2=01;31:*.tz=01;31:*.deb=01;31:*.rpm=01;31:*.jar=01;31:*.war=01;31:*.ear=01;31:*.sar=01;31:*.rar=01;31:*.alz=01;31:*.ace=01;31:*.zoo=01;31:*.cpio=01;31:*.7z=01;31:*.rz=01;31:*.cab=01;31:*.wim=01;31:*.swm=01;31:*.dwm=01;31:*.esd=01;31:*.jpg=01;35:*.jpeg=01;35:*.mjpg=01;35:*.mjpeg=01;35:*.gif=01;35:*.bmp=01;35:*.pbm=01;35:*.pgm=01;35:*.ppm=01;35:*.tga=01;35:*.xbm=01;35:*.xpm=01;35:*.tif=01;35:*.tiff=01;35:*.png=01;35:*.svg=01;35:*.svgz=01;35:*.mng=01;35:*.pcx=01;35:*.mov=01;35:*.mpg=01;35:*.mpeg=01;35:*.m2v=01;35:*.mkv=01;35:*.webm=01;35:*.webp=01;35:*.ogm=01;35:*.mp4=01;35:*.m4v=01;35:*.mp4v=01;35:*.vob=01;35:*.qt=01;35:*.nuv=01;35:*.wmv=01;35:*.asf=01;35:*.rm=01;35:*.rmvb=01;35:*.flc=01;35:*.avi=01;35:*.fli=01;35:*.flv=01;35:*.gl=01;35:*.dl=01;35:*.xcf=01;35:*.xwd=01;35:*.yuv=01;35:*.cgm=01;35:*.emf=01;35:*.ogv=01;35:*.ogx=01;35:*.aac=00;36:*.au=00;36:*.flac=00;36:*.m4a=00;36:*.mid=00;36:*.midi=00;36:*.mka=00;36:*.mp3=00;36:*.mpc=00;36:*.ogg=00;36:*.ra=00;36:*.wav=00;36:*.oga=00;36:*.opus=00;36:*.spx=00;36:*.xspf=00;36:
DEFAULT_APP_ID=app-id
DEFAULT_APP_KEY=app-key
SOKETI_DB_MYSQL_USERNAME=mydbuser
SSH_CONNECTION=10.168.165.18 56571 192.168.100.110 22
SOKETI_APP_MANAGER_MYSQL_USE_V2=false
SOKETI_APP_MANAGER_MYSQL_TABLE=apps
LESSCLOSE=/usr/bin/lesspipe %s %s
XDG_SESSION_CLASS=user
TERM=xterm-256color
LESSOPEN=| /usr/bin/lesspipe %s
USER=localadmin
SHLVL=1
XDG_SESSION_ID=1
LC_CTYPE=C.UTF-8
XDG_RUNTIME_DIR=/run/user/1000
SSH_CLIENT=10.168.165.18 56571 22
XDG_DATA_DIRS=/usr/local/share:/usr/share:/var/lib/snapd/desktop
SOKETI_APP_MANAGER_MYSQL_VERSION=8
PATH=/usr/local/sbin:/usr/local/bin:/usr/sbin:/usr/bin:/sbin:/bin:/usr/games:/usr/local/games:/snap/bin
DBUS_SESSION_BUS_ADDRESS=unix:path=/run/user/1000/bus
SSH_TTY=/dev/pts/0
test=test
OLDPWD=/home
_=/usr/bin/env
